assignment-6&7/src/Components/CardContainer.js

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom';
+
+const CardComponent = ({ singleUser }) => {
+    const { login, name, avatar_url } = singleUser;
```

Comment:
We can destructure id as well here 
const { login, name, avatar_url  , id} = singleUser;
and use it below rather than writing key={singleUser.id} in the div

Reply:
yes can destructure further to something like this:
const CardComponent = ({ singleUser : { login, name} }) => {